src/main/java/com/google/api/codegen/util/ruby/RubyNameFormatter.java

```diff
@@ -17,6 +17,7 @@
 import com.google.api.codegen.util.Name;
 import com.google.api.codegen.util.NameFormatter;
 import com.google.api.codegen.util.NamePath;
+import com.google.common.collect.ImmutableSet;
 
 /**
  * The NameFormatter for Ruby.
```

Comment:
I was just skipping this file. Added (note that skipping private fields intentionally because those are `@foo` style IIRC).